src/rendering/renderers/ActualPriceLineRenderer.ts

```diff
@@ -0,0 +1,110 @@
+import config from '../../config'
+
+import { Graphics } from '../../lib/pixi'
+import datamath from '../../lib/datamath'
+
+import { IGraphicRenderer, RenderingContext } from '..'
+import { BaseRenderer, GraphicUtils } from '..'
+
+export class ActualPriceRenderer extends BaseRenderer {
+
+    static readonly ACTUAL_PRICE_ID: symbol = Symbol('ACTUAL_PRICE_ID')
```

Comment:
It's not really actual price, it's just latest price we have for now
So rename it to latest price
Rename `ActualPriceRenderer` => `LatestPriceRenderer`
Rename `ACTUAL_PRICE_ID` => `LATEST_PRICE_ID`
...